docs/adr/021-pagination-filtering-on-frontend.md

```diff
@@ -0,0 +1,40 @@
+# 21. Pagination/Filtering On Frontend
+
+Date: 2023-10-24
+
+## Status
+Accepted
+
+## Context
+
+Establish any guidelines to help in determining whether we should load all the BLIs into Redux in one backend call or whether to make multiple paginated calls to the API.
+
+If it is possible to load all the BLIs at once there are at least 2 advantages:
+
+1. Filtering/Sorting/Searching would be fast
+2. There would be less calls to the API (reduced load and latency)
+
+The concern is whether a representative data set would fit into Redux without problems and how long the initial load of the BLIs would take.
+
+## Decision
+
+John DeAngelis conducted performance tests documented here: https://docs.google.com/document/d/1hMxrv-Zhv_xu2E0tTMzRdNTiD1M-GjCg1x2tVVJ4DKI/edit?usp=sharing
```

Comment:
wondering about the share-ability of this document? Could folks outside of Flexion see the results?

Reply:
good question, was almost thinking to maybe export to pdf and file in the repo somewhere? or convert the report to `md` and include at the bottom of this file.

I don't think anything sensitive in nature is exposed in the contents right?

Reply:
Agree, a simple MD conversion would be nice for this.